src/subcommand/wallet/inscribe.rs

```diff
@@ -46,6 +46,8 @@ pub(crate) struct Inscribe {
   pub(crate) no_backup: bool,
   #[clap(long, help = "Don't sign or broadcast transactions.")]
   pub(crate) dry_run: bool,
+  #[clap(long, help = "Send reveal output to <INSCRIPTION_DESTINATION>")]
+  pub(crate) inscription_destination: Option<Address>,
```

Comment:
```suggestion
  #[clap(long, help = "Send inscription to <DESTINATION>.")]
  pub(crate) destination: Option<Address>,
```

Reply:
I suck at naming things. thanks for the suggestion!